pygenesys/tests/test_db_creator.py

```diff
@@ -0,0 +1,73 @@
+from pygenesys.db_creator import *
+import os
+
+curr_dir = os.path.dirname(__file__)
+test_db = curr_dir + '/test_db.sqlite'
```

Comment:
was this sqlite db generated using `test_inputfile.py`?

Reply:
nope, this is strictly for unit testing the functions in db_creator.py. This database is created and removed within the tests. Perhaps there's a better way, I didn't think too hard about it.